wykop/api/client.py

```diff
@@ -28,8 +29,12 @@ def request(self, rtype, rmethod=None,
                                       post_params=post_params,
                                       file_params=file_params)
 
-    def authenticate(self, account_key=None):
-        self.requestor.authenticate(account_key)
+    def authenticate(self, account_key=None, login=None, password=None):
```

Comment:
Może warto byłoby tutaj sprawdzić czy podano dobrą kompinację argumentów? Tj albo samo `account_key` albo `login` i `password`. Tak żeby chronić później użytkownika przed własną głupotą.

Reply:
Jeśli nie tu to może wewnątrz `requestor.authenticate`

Reply:
> Może warto byłoby tutaj sprawdzić, czy podano dobrą kombinację argumentów? Tj albo samo `account_key` albo `login` i `password`. Tak, żeby chronić później użytkownika przed własną głupotą.

jeśli ma się pojawić możliwość logowania przez login i hasło na nieoficjalnych kluczach, to zadziała w taki sposób, że będzie na podstawie loginu i hasła zdobywać accountkey, więc mało przyszłościowe wymaganie